annotation_tracker/web_client/panels/Experiments/index.js

```diff
@@ -12,21 +13,53 @@ const Experiments = Panel.extend({
     }),
 
     initialize() {
-        console.log('Experiments.initialize()');
-
         this.running = false;
-        this.experiment = 0;
-        this.task = 0;
+        this.current_experiment  = 0;
+        this.current_task = 0;
+        this.experiment = 'Loading...';
+        this.task = '';
+        this.complete = false;
+        this.experiments = null;
+    },
+
+    setFolderId(folderId) {
+        //fetch the metadata from the FolderID
+        const folderModel = new FolderModel();
+        folderModel.set({
+            _id: folderId
+        }).on('g:fetched', () => {
+            this.meta = folderModel.get('meta');
+            this.processMetadata(this.meta);
+        }).on('g:error', () => {
+            console.warn(`Error fetching metadata for folder: ${folderId}`);
+        }, this).fetch();
+    },
+
+    processMetadata(metadata) {
+        if (metadata["Experiments"] !== undefined) {
+            // TODO: JSON Schema validation at some point to ensure we have all necessary data
+            this.current_experiment  = 0;
+            this.current_task = 0;    
+            this.experiments =  metadata["Experiments"];
```

Comment:
You can use `metadata.Experiments` instead of `metadata["Experiments"]`, which is usually preferred by linters.